### ISSUE TITLE
Query parameters

### ISSUE THREAD

**Issue by Jimflip** [trellis-ldp/trellis#1667]
Does this support querying a resource type i.e. class and providing some query parameters to filter the result?
Or can you only get an explicit resource by id, and all resources by Class?

I ask as I don't see any mention of this in Trellis or LDP docs.

Thanks.

**Comment by acoburn** [comment]
No, Trellis does not support query, only `GET` on individual resources. This is in line with LDP.

What you are likely interested in is indexing resources into a triple store or some external index, possibly via the [notification engine](https://github.com/trellis-ldp/trellis/wiki/Notifications) that Trellis does support. Once indexed, you can define whatever sort of query API you wish to support.
